### DISCUSSION TITLE
[2024-10-11] AMER Building DevRel Foundation Community Call

### DISCUSSION THREAD

**Discussion by jcleblanc** [DevRel-Foundation/governance#110] [Meeting Notes]
> Part of our community calls are being used as Open Office Hours to reach community consensus on Foundation Next Steps. If you would like to get involved, this is a safe and welcoming space to start!

> Whether you are new to the call or a regular participant, you can add your discussion topics as a comment below or bring them up during the call. You can also share DevRel use cases, best practices, frameworks, and more by opening a new issue in the [GitHub Discussions](https://github.com/DevRel-Foundation/governance/blob/main/working-groups/themes.md)), which serves as the Foundation’s public forum. Additionally, we are always looking for 5-10 minute presentations in future community calls to spark discussion with the rest of the attendees and async.

## Agenda

👋 Intros  - If we have a large group, we'll use chat
🧹 Housekeeping
- Code of conduct ([link](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md)). 
Instances of abusive, harassing, or otherwise unacceptable behavior, please report at conduct@dev-rel.org
- Intro to roles: facilitator, moderator, scribe

🤔 What we're doing in this call

The Developer Relations Foundation mission:
_The mission of the Developer Relations Foundation project is to elevate the professional practice of developer relations and increase awareness of it as a driver of business value_

### Open Office Hours

Time to listen to the voices of community members, answer questions, and tackle blockers together. Topics can be related to:

- Update on current [phased timeline](https://github.com/orgs/DevRel-Foundation/projects/7/views/1) - where we are and what to expect. As part of next steps / expectations, getting people signed up for the working groups and management spots.

- Walkthrough and discussion of the [final themes and groupings](https://github.com/DevRel-Foundation/governance/blob/main/working-groups/themes.md). Collect any feedback on missing items or adjustments.

- Hey YOU! 🙋🏽‍♀️🙋🏽🙋🏻🙋🏼‍♀️🙋🏻 Whether you are new to the call or a regular participant, the community shapes the conversations happening in DevRel Foundation calls! Please add your discussion topics as a comment below or bring them up during the call. You can also share any DevRel use cases, best practices, frameworks, etc as a forum, by opening a new issue in the GH Discussions.

## Essential Information

* 🕛 Friday October 11th, 12:00 - 1:00 pm EST
* 📝 [g.doc agenda](https://docs.google.com/document/d/1I5dX4RbvJyt4Z2dTWsS1BjmzRQV8RHOJ5klJ_tZa_eg/edit?usp=sharing)

| Frequency | Duration | Zoom link | Join Discord |
| -- | -- | -- | -- |
| bi-weekly | 1 hour | [Here](https://zoom.us/j/97083888502?pwd=5WQDWGN7U08Nvw6r12nQB48Kc5hqg2.1) | [Join](https://discord.gg/2jZ833bx) |

**Comment by katiewaz1977** [comment]
**Attendees**
Jaime Lopez
Stacey Kruczek
Katie Miller

**Meeting notes**

- Phased timeline - link is locked, need to fix that (FIXED)
- Appreciate the care of moving things forward carefully
- Potential worry / angst around certification - will this work be a gatekeeper for folks who wish to be in this profession but don’t complete a certification?
